### PR TITLE
chore(readme): update examples of conference app to include all frameworks

### PR DESCRIPTION
**Pull request by brandyscarney**
I noticed that we are missing the links to the react & vue conference apps so I added them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,8 +77,11 @@ Already have an Ionic app? These guides will help you migrate to the latest vers
 
 ### Examples
 
-The [Ionic Conference App](https://github.com/ionic-team/ionic-conference-app) is a full featured Ionic app.
-It is the perfect starting point for learning and building your own app.
+The Ionic Conference App is a full featured Ionic app. It is the perfect starting point for learning and building your own app.
+
+- [Angular Ionic Conference App](https://github.com/ionic-team/ionic-conference-app)
+- [React Ionic Conference App](https://github.com/ionic-team/ionic-react-conference-app)
+- [Vue Ionic Conference App](https://github.com/ionic-team/ionic-vue-conference-app)
```
[the remark this answers is not in the document]
Comment (brandyscarney) [reply README.md:84]: I can remove it, should I create an issue somewhere to fix it up + link to it once that's done?

[the remark this answers is not in the document]
Comment (liamdebeasi) [reply README.md:84]: Yeah we should have a tech debt ticket for updating the Vue conf app to vue 3 + latest Ionic

[the remark this answers is not in the document]
Comment (brandyscarney) [reply README.md]: I added a tech debt ticket and linked to it in the readme

**Review by liamdebeasi — APPROVED** [review]
Thanks!

**Pull request merged**